Core/src/IfcGeometryConverter/ProfileConverter.h

```diff
@@ -1522,6 +1477,64 @@ namespace OpenInfraPlatform {
 				return paths;
 			}
 
+			// Function 10: Get rectangle coordiantes
+			
+			void GetRectangleCoordinates (std::vector<carve::geom::vector<2>>& coords, double radius, double x, double y) const

```

Comment:
Also, about naming: the function *adds* coordinates to `coords` - I would call it AddRectangleCoordinates.
```suggestion
			void AddRectangleCoordinates (std::vector<carve::geom::vector<2>>& coords, double radius, double x, double y) const

```

Reply:
Done